legion/engine/physics/systems/physicssystem.hpp

```diff
@@ -237,7 +273,14 @@ namespace legion::physics
             }
 
             
+            for (auto& manifold : manifoldsToSolve)
+            {
+                log::debug("----> Final Velocity");
+                //manifold.contacts.at(0).logRigidbodyState();
+            }
+            
```

Comment:
Not only is this a debug, but it is a looped debug. This can possibly be removed